packages/react-native-bpk-component-button/stories.js

```diff
@@ -38,6 +38,10 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
     flexWrap: 'wrap',
     justifyContent: 'flex-start',
+    padding: spacingMd,
+  },
+  btnContainerDark: {
+    backgroundColor: colorAbisko,
```

Comment:
nit: what about using black so we keep a dark background in dark mode? 👼

Reply:
But then I'd be using black in light mode too, which is against our brand 😉